core/tests/puppeteer-acceptance-tests/utilities/user/blog-post-editor.ts

```diff
@@ -145,22 +149,73 @@ export class BlogPostEditor extends BaseUser {
   }
 
   /**
+   * This is a composite function that can be used when a straightforward, simple blog published is required.
    * This function publishes a blog post with given title.
    */
   async publishNewBlogPostWithTitle(newBlogPostTitle: string): Promise<void> {
-    await this.addUserBioInBlogDashboard();
-    await this.clickOn(LABEL_FOR_NEW_BLOG_POST_CREATE_BUTTON);
+    await this.navigateToBlogEditorPage();
+    await this.uploadBlogPostThumbnailImage();
     await this.expectPublishButtonToBeDisabled();
 
-    await this.uploadBlogPostThumbnailImage();
+    await this.updateTitleTo(newBlogPostTitle);
+    await this.updateBodyTextTo('test blog post body content');
+    await this.selectTags('News', 'International');
+    await this.saveTheChanges();
+
+    await this.publishTheBlogPost();
+  }
+
+  /**
+   * This function navigates to the blog editor page.
+   */
+  async navigateToBlogEditorPage(): Promise<void> {
```

Comment:
I'm confused about this function. The name means it's a function to navigate to Blog Editor Page but you are adding a user bio, creating a blog, and verifying if a publish button is being disabled? I don't see any navigation part.

Reply:
@imchristie 
After we reach `http://localhost:8181/blog-dashboard` page then to create a blog we need to add our user name and bio then click on `CREATE NEW BLOG POST` button to reach actual blog editor where we can create a blog.
i.e Steps to go create blog after reaching blog-dashboard -->

- Add username and bio
- Click on `CREATE NEW BLOG POST`
- Once we reach the blog editor, the publish button should be disabled since we have not filled in any entry.

Reply:
corrected to `openBlogEditorPage`